apps/journey/app/utils/validateLanguage.ts

```diff
@@ -0,0 +1,10 @@
+export const supportedLanguages = ['en', 'ga'];
+export const defaultLanguage = 'en';
+
+export default function validateLanguage(lang: string): "en" | "ga"  {
+    if (supportedLanguages.includes(lang)) {
+        return lang as "en" | "ga";
```

Comment:
Fancier if you like it:

```suggestion
        return lang as unknown as (typeof supportedLanguages)[number];
```